### ISSUE TITLE
"DisableBrokenFilter03" aka the yellow tint doesn't work anymore on 1.9.1

### ISSUE THREAD

**Issue by SixelAlexiS90** [nipkownix/re4_tweaks#532]
### Validation

- [X] Game has been updated to the latest RE4HD release (v1.1: https://www.re4hd.com/?p=9552)
- [X] Data has been [validated with SFV](https://www.re4hd.com/?p=9454) (a guide to using QuickSFV is available at that link, use that with this 1.1 SFV: [BIO4-HDProject1.1-SFV.zip](https://github.com/emoose/re4-research/files/9207370/BIO4-HDProject1.1-SFV.zip))

### Describe your issue here (drag+drop ZIP to attach it)

As the title say, the "DisableBrokenFilter03" aka the yellow tint doesn't work anymore on 1.9.1, it seems that it's disabled by default and you can't enable it again even if you want.

**Comment by Klemci** [comment]
Ttry that :

ORIGINAL AND HD PROJECT COLOR GRADIENT FILES
https://www.re4hd.com/?page_id=9322

**Comment by nipkownix** [comment]
That option is forcefully disabled if the HD Project is detected, since it is necessary. Do you happen to be using the HD Project?

**Comment by SixelAlexiS90** [comment]
> That option is forcefully disabled if the HD Project is detected, since it is necessary. Do you happen to be using the HD Project?

Yes I'm using HD Project but I've tried it with the original version as well and the filter is always disabled no matter what.
It used to work on 1.8.

**Comment by nipkownix** [comment]
Are you trying to _enable_ that option? If you're using the HD Project, you just won't be able to. It may have worked in the past since we used another method to detect whether or not the HD Project was installed, but we've recently updated it to make sure the Project settings are properly applied.

Besides, the filter doesn't make the image orange with the HD Project. It was repurposed to do something else, which is why we need the filter _enabled_, thus having the option _disabled_.

Now, if you're trying to enable the option in the vanilla game and still can't, please upload a dinput8.log here.

**Comment by SixelAlexiS90** [comment]
> Are you trying to _enable_ that option? If you're using the HD Project, you just won't be able to. It may have worked in the past since we used another method to detect whether or not the HD Project was installed, but we've recently updated it to make sure the Project settings are properly applied.
> 
> Besides, the filter doesn't make the image orange with the HD Project. It was repurposed to do something else, which is why we need the filter _enabled_, thus having the option _disabled_.
> 
> Now, if you're trying to enable the option in the vanilla game and still can't, please upload a dinput8.log here.

Yeah, it doesn't work on vanilla game, it's like the option is always checked (and so the filter is always removed). Here the log:
[dinput8.log](https://github.com/nipkownix/re4_tweaks/files/11125799/dinput8.log)
